components/apps_form/apps_form_component.scss

```
@@ -19,3 +19,42 @@
 .apps-form-modal-body-loading {
     z-index: 1000;
 }
+
+.select_option_item {
+display: flex;
+    flex-direction: row;
+    align-items: center;
+    column-gap: 1.2rem;
+}
```

We need to make sure these styles are inside of a selector specific to the Apps form. We can use the `#appsModal` selector like so:

```suggestion
#appsModal {
    .select_option_item {
        display: flex;
        flex-direction: row;
        align-items: center;
        column-gap: 1.2rem;
    }
}
```